pertpy/tools/_metadata/_cell_line.py

```diff
@@ -258,14 +260,23 @@ def annotate_cell_lines(
                 )
 
             if len(not_matched_identifiers) > 0:
-                print(
-                    f"[bold blue]There are {identifier_num_all} identifiers in `adata.obs`."
-                    f"However, {len(not_matched_identifiers)} identifiers can't be found in the cell line annotation,"
-                    "leading to the presence of NA values for their respective metadata.\n",
-                    "Please check again: ",
-                    *not_matched_identifiers,
-                    sep="\n- ",
-                )
+                if isinstance(show, str):
+                    if show != "all":
+                        raise ValueError("Only a non-positive value or all is accepted.")
```

Comment:
```suggestion
                        raise ValueError("Only a non-negative value or 'all' is accepted.")
```

non-negative, right?